tests/vmi_multus_test.go

```diff
@@ -1240,3 +1244,35 @@ func validatePodKubevirtResourceName(virtClient kubecli.KubevirtClient, vmiPod *
 
 	return nil
 }
+
+func defaultCloudInitNetworkData() string {
+	networkData, err := libnet.CreateDefaultCloudInitNetworkData()
+	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "should success creating default cloud init network data for SRIOV")
```

Comment:
nit: "should succeed in creating..."
or you can: "should successfully create..."

Reply:
Done